pyqg/model.py

```diff
@@ -664,6 +664,36 @@ def parameterization_spectrum(m):
             dims=('l','k')
         )
 
+        def dissipation_spectrum(m):
+            spectrum = np.zeros_like(m.qh)
+            ones = np.ones_like(m.filtr)
+            if m.ablevel==0:
+                # forward euler
+                dt1 = m.dt
+                dt2 = 0.0
+                dt3 = 0.0
+            elif m.ablevel==1:
+                # AB2 at step 2
+                dt1 = 1.5*m.dt
+                dt2 = -0.5*m.dt
+                dt3 = 0.0
+            else:
+                # AB3 from step 3 on
+                dt1 = 23./12.*m.dt
+                dt2 = -16./12.*m.dt
+                dt3 = 5./12.*m.dt
+            for k in range(m.nz):
+                spectrum[k] += (m.filtr - ones) * (
+                    m.qh[k] + dt1*m.dqhdt[k] + dt2*m.dqhdt_p[k] + dt3*m.dqhdt_pp[k])
+            return -np.real(np.tensordot(m.Hi, np.conj(m.ph) * spectrum, axes = (0, 0))) / m.H / m.dt
+
+        self.add_diagnostic('Dissspec',
+            description='Spectral contribution of filter dissipation',
+            function=dissipation_spectrum,
+            units='',
```

Comment:
Why are the units blank?

Reply:
Added units for Dissspec ("meters squared second ^-3", consistent with the `diagnostics-units` branch)